docs/next/guides/formulas/formula-calculation.md

```diff
@@ -3,196 +3,435 @@ title: Formula calculation
-* Custom variables
-* Nested functions
-* Dynamic updates
+*   Fully-featured spreadsheet apps
+*   Smart documents
+*   Educational apps
+*   Business logic builders
+*   Forms and form builders
+*   Online calculators
+*   Low connectivity apps
+
+## Features
+
+*   High-speed formula calculations
+*   Function syntax compatible with Excel and Google Sheets
+*   A library of built-in functions available in 16 languages
+*   Support for wildcard characters
+*   Support for CRUD operations
+*   Support for cross-sheet references
+*   Support for multiple Handsontable instances
+*   Uses GPU acceleration for better performance
 
 **Known limitations:**
 
-* Not working with filtering and column sorting;
-* Not working with trimming rows.
+*   Doesn't work with nested rows
+*   Doesn't work with undo/redo
+
+## Available options and methods
+
+For the list of available settings and methods, visit the [API reference](@/api/formulas.md).
+
+## Available functions
+
+This plugin inherits the calculation powers from _HyperFormula_. The complete functions reference can be found in the [HyperFormula documentation](https://handsontable.github.io/hyperformula/guide/built-in-functions.html).
 
-## Quick setup
+## Basic multi-sheet example
 
-To enable the plugin you need to set the `formulas` property to `true`. Cells that are dependent on the edited value will be dynamically recalculated.
+It is possible to use the plugin in single sheet mode or with multiple Handsontable instances with cross-sheet references.
 
-::: example #example1
+Double click on a cell to open the editor and preview the formula.
+
+::: example #example1 --html 1 --js 2
+```html
+<div id="example-basic-multi-sheet-1"></div>
+<div id="example-basic-multi-sheet-2"></div>
+```
 ```js
-const container = document.querySelector('#example1');
+const data1 = [
+  ['10.26', null, 'Sum', '=SUM(A:A)', null],
+  ['20.12', null, 'Average', '=AVERAGE(A:A)', null],
+  ['30.01', null, 'Median', '=MEDIAN(A:A)', null],
+  ['40.29', null, 'MAX', '=MAX(A:A)', null],
+  ['50.18', null, 'MIN', '=MIN(A1:A5)', null],
+];
 
-const data = [
-  ['=$B$2', 'Maserati', 'Mazda', 'Mercedes', 'Mini', '=A$1'],
-  [2009, 0, 2941, 4303, 354, 5814],
-  [2010, 5, 2905, 2867, '=SUM(A4,2,3)', '=$B1'],
-  [2011, 4, 2517, 4822, 552, 6127],
-  [2012, '=SUM(A2:A5)', '=SUM(B5,E3)', '=A2/B2', 12, 4151]
+const data2 = [
+  ['Is A1 in Sheet1 > 10?', '=IF(Sheet1!A1>10,"TRUE","FALSE")', null, null],
+  ['Is A:A in Sheet > 150?', '=IF(SUM(Sheet1!A:A)>150,"TRUE","FALSE")', null, null],
+  ['How many blank cells are in the Sheet1?', '=COUNTBLANK(Sheet1!A1:E5)', null, null],
+  ['Generate a random number', '=RAND()', null, null],
+  ['Number of sheets in this workbook', '=SHEETS()', null, null],
 ];
 
-const hot = new Handsontable(container, {
-  data,
+const hyperformulaInstance = HyperFormula.buildEmpty();
+
+const container1 = document.getElementById('example-basic-multi-sheet-1');
+new Handsontable(container1, {
+  data: data1,
```

Comment:
There is lack of `licenseKey: 'non-commercial-and-evaluation',`. Could you please add it?

Reply:
d4905055c0f6487bfc107b5ff3b7a6bb796193ab